VisualRust.Test/VisualRust.Test.csproj

```diff
@@ -117,7 +124,9 @@
       <Name>VisualRust.Project</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
Visual Studio continuesly add this line. There some disscusstions about this line https://github.com/nunit/nunit-vs-adapter/issues/51 http://stackoverflow.com/questions/18614342/what-is-service-include-in-a-csproj-file-for. We can remain this line or change project type.